README.md

```diff
@@ -18,9 +18,9 @@ This site or product includes IP2Location LITE data available from <a href="http
 
 ```js
 // Setup
-const CountryLookup = require('ip3country');
+const IP3Country = require('ip3country');
```

Comment:
should use the same casing

Reply:
But it's a class though, what takes precedence?